### PR TITLE
Migration of the solution to CDK V2 and relevant upgrades

### PR DESCRIPTION
**Pull request by velartur**
*Issue #, if available:*
It is not possible to deploy the solution because CDK V1 is depreciated.

*Description of changes:*
The changes are mainly made to migrate from CDK V1 to CDK V2, also the compilation of the clips app is updated to fix a bug with version 1.21 of the Go runtime. Finally the IVS SDK for the player is updated to version 1.23.0.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: cdk/Makefile [hunk 1]
```diff
@@ -8,10 +8,10 @@ cdksynth: buildcdk
 	npx cdk -v synth ClipsUI
 
 deploy: build
-	npx cdk -v deploy ClipsBackend --outputs-file tempvars.json
+	cdk deploy -v --outputs-file tempvars.json ClipsBackend
 	node repack.js
 	rm tempvars.json
-	npx cdk -v deploy ClipsUI
+	cdk deploy -v ClipsUI
```
Comment (andreido) [inline cdk/Makefile:14]: To remain consistent with our other IVS demos, please run the CDK deploy scripts using `npx`.

File: web-ui/index.html [hunk 2]
```diff
@@ -12,7 +12,7 @@
   <title>Amazon IVS - Clip demo</title>
 
   <link rel="stylesheet" href="style.css">
-  <script src="https://player.live-video.net/1.8.0/amazon-ivs-player.min.js"></script>
+  <script src="https://player.live-video.net/1.23.0/amazon-ivs-player.min.js"></script>
```
Comment (andreido) [inline web-ui/index.html:15]: The latest version of the Player SDK is [1.24.0](https://docs.aws.amazon.com/ivs/latest/LowLatencyUserGuide/release-notes.html#jan18-24) - please upgrade to this version.

File: cdk/Makefile (hunk 1 above)
Comment (velartur) [reply cdk/Makefile]: scripts homologated with npx ;)

File: web-ui/index.html (hunk 2 above)
Comment (velartur) [reply web-ui/index.html]: updated Player SDK version to 1.23.0

**Comment by andreido** [comment]
Thank you for the contribution @velartur!

**Pull request merged**